lib/smart_proxy_dynflow/action/shareable.rb

```diff
@@ -12,14 +12,14 @@ def plan(input)
       planned_action = plan_self(input)
       # code only applicable, when run with SmartProxyDynflowCore in place
       if on_proxy? && callback
-        plan_action(SmartProxyDynflowCore::Callback::Action, callback, planned_action.output)
+        plan_action(Proxy::Dynflow::Callback::Action, callback, planned_action.output)
       end
     end
 
     private
 
     def on_proxy?
-      defined?(SmartProxyDynflowCore::Callback)
+      defined?(Proxy::Dynflow::Callback)
     end
```

Comment:
This should now always be true, the action cannot run anywhere else than on the smart proxy

Reply:
Do we want to remove that then and remove usage of it?
Or just return always `true` and have a followup?

Reply:
I feel the latter is what we want, so I went with it, I'll try to come up with follow-up

Reply:
It doesn't seem to be used outside of this repository so I'd be fine with removing it along with its usage in this pr

Reply:
Hah, github didn't refresh. Nevermind then, let's do it in a followup